src/test/ui/consts/rfc-2203-const-array-repeat-exprs/migrate-const-fn-pass.rs

```diff
@@ -0,0 +1,128 @@
+// compile-pass
+// compile-flags: -Z borrowck=migrate
+// ignore-compare-mode-nll
+#![feature(const_in_array_repeat_expressions)]
+#![allow(warnings)]
+
+// Some type that is not copyable.
+struct Bar;
+
+mod constants {
+    use Bar;
+
+    const fn no_impl_copy_empty_value_no_elements() {
```

Comment:
Oh... hmm; I suppose I should have been clearer... =)

What I meant is calling a `const fn` for the element expression.

Reply:
Fixed.